### PR TITLE
[DCOS-51986] Updated attribute type to 'media' for service account and secret

### PR DESCRIPTION
**Pull request by rathohit311356**
## What changes were proposed in this pull request?
Added Media Attribute change for Service Account and Secret , post this change autosuggestion will appear as drop down for the service account and secret present on the cluster. 

Resolves [DCOS-51986: Add appropriate media attribute for secrets and service accounts to json ](https://jira.mesosphere.com/browse/DCOS-51986)

## How were these changes tested?
Changes were tested on CCM cluster by locally building the stub , screen shot for the same been attached to JIRA ticket and also attached to this PR

## Release Notes

Enabled Autocomplete widgets for Service Account and Secret , Enables users to quickly find and select from a pre-populated list of values.

Screen shot attached showing how service account "spark" and secret "spark-secret" getting pre-populated

spark-service-account
spark-secret

### CODE REVIEW

**Comment by rathohit311356** [comment]
@akirillov Thanks Anton for review , i have updated as per your comment , let me know if it looks good now.

File: universe/config.json [hunk 1]
```diff
@@ -31,12 +31,18 @@
...
+                    }
                 },
                 "service_account_secret": {
                     "description": "The Spark Dispatcher will register with mesos with this secret.",
                     "type": "string",
-                    "default": ""
+                    "default": "",
+                    "media": {
+                       "type": "application/x-secret+string"
+                     }
```
Comment (samvantran) [inline universe/config.json]: formatting slightly off here

Comment (rathohit311356) [reply universe/config.json]: Fixed

**Comment by rathohit311356** [comment]
@samvantran @akirillov  Hi - Guys , Do let me know if you see any other concern on this PR.

**Pull request merged**